asmcli/lib/util.sh

```diff
@@ -0,0 +1,22 @@
+# expects var PARAM already defined and populates it with the
+# appropriate manifest overlays
+init_install_params() {
```

Comment:
I would much rather this `echo` the params at the end, and then call it with `PARAMS="$(gen_install_params)"`

Even better would be to have an inner function which took the CA and istioctlFiles as arguments with the actual logic, so that we can inject test params in the inner function and still be able to use the no-argument version for actual client logic.

Reply:
made the former modification. i can refactor this into more decouple functions in the next pass to simplify unit testing.